### ISSUE TITLE
error:  Unexpected token

### ISSUE THREAD

**Issue by czz362100** [TyrealHu/acorn-typescript#32]
hey, team. i had some questions for this plugin

parsefile's  code
```javascript
const testApp = async(app: string, index: number) => {
};
```
parseFunction
```javascript
export class ScanAnalyzer{
  private parser: any;

  constructor() {
    this.parser = Parser.extend(
      tsPlugin({
        jsx: {
          allowNamespaces: true,
        },
      })
    );
  }
  scanFileForChinese(filePath: string) : boolean {
    const sourceCode = fs.readFileSync(filePath[0], "utf8");

    const ast = this.parser.parse(sourceCode, {
      ecmaVersion: "latest",
      sourceType: "module",
      locations: true,
      onComment: (isBlock, text, start, end, startLoc, endLoc) => {
        if (isBlock) {
          console.warn(
          
          );
        } else {
          console.warn(
           
          );
        }
      }, // 保留注释信息
    });

    let hasChinese = false;
  
    
    walk.simple(
      ast,
      {
        Literal(node) {
          if (typeof node.value === "string" && chineseRegExp.test(node.value)) {
            console.warn(
             
            );
            hasChinese = true;
          }
        },
        VariableDeclarator(node) {
          if (node.init === null) {
            console.warn(
              `Variable ${node.id.name} at line ${node.loc.start.line} has a missing initializer.`
            );
          }
        },
      },
      {
        ...walk.base,
        JSXElement: (node) => {
          console.log(node)
        }
      }
    );
    return hasChinese
  }

  scanFilesForChinese(files: string[]) {
    let hasChinese = false;

    files.forEach((file) => {
      if (this.scanFileForChinese(file)) {
        hasChinese = true;
      }
    });
  
    return hasChinese;
  }
}
```

why show this error: SyntaxError: Unexpected token (1:23)

**Comment by TyrealHu** [comment]
What version did u use?

**Comment by czz362100** [comment]
my version
<img width="269" alt="image" src="https://github.com/TyrealHu/acorn-typescript/assets/37129738/beafe4be-90a9-4bb2-ae0a-da03dae9d2a7">

**Comment by TyrealHu** [comment]
this issue has been fixed in acorn-typescript@1.4.1

**Comment by czz362100** [comment]
ok. tks so mush. it work.
